src/bin/big_context_no_loop.rs

```diff
@@ -0,0 +1,104 @@
+extern crate tera;
```

Comment:
is this file needed? can it be in bench instead?

Reply:
Removed file.